### DISCUSSION TITLE
Abrir el último proyecto

### DISCUSSION THREAD

**Discussion by SirRickster** [gusmanb/ZXBasicStudio#24] [Ideas]
Una opción en el menú file que directamente abra el último proyecto en el que se trabajó, para no tener que ir a buscarlo.

Sería buena idea guardar la ruta y nombre de la carpeta en el fichero de opciones/configuración una vez cerramos el proyecto o salimos de la app?

**Comment by gusmanb** [comment]
Estaría bien crear una lista de los últimos proyectos abiertos y de los últimos archivos abiertos y se guardan en una lista, los 10 últimos por ejemplo y añadir un "Open recent files" y "Open recent projects" y dentro aparezca el listado, que te parece?

**Reply by SirRickster** [reply]
Bueno, yo era por hacerlo sencillo , pero tu opción también es válida :)

**Reply by gusmanb** [reply]
Si quieres se puede implementar de momento la "opción rápida" y dejar solo el último proyecto, y luego ya metemos el listado.

**Reply by SirRickster** [reply]
Pues este fin de semana le doy una vuelta :)
